service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -157,4 +163,12 @@ private String concatGatewayConfigurationUrlAndMostRecentUploadedOrderDocumentPa
         }
         return "";
     }
+
+    private DocmosisTemplates getDocmosisTemplateType(GeneratedOrderType type) {
+        if (type.equals(EMERGENCY_PROTECTION_ORDER)) {
```

Comment:
if you want to use `.equals` it is better practice to do `ENUM.equals(variable)` but with enums you can use `==` which is null safe by default. if you want to simplify this even further you can do `return type == EMERGENCY_PROTECTION_ORDER ? EPO : ORDER`